lib/Bindings/Python/circt/dialects/_hw_ops_ext.py

```diff
@@ -197,7 +197,7 @@ def _create_output_op(cls_name, output_ports, entry_block, bb_ret):
   # Only acceptable return is a dict of port, value mappings.
   if not isinstance(bb_ret, dict):
     raise support.ConnectionError(
-        "In {cls_name}, can only return a dict of port, value mappings "
+        f"In {cls_name}, can only return a dict of port, value mappings "
         "from body_builder.")
```

Comment:
nit: does this line need any extra space of formatting :thinking:  fyi the format check is now working on main so we may need to re-run yapf to clean up anything that slipped by.

Reply:
> extra space of formatting

Huh? Yapf doesn't complain about this line.

Reply:
And yeah... I'll re-run yapf and do a commit.

Reply:
I was just eyeballing it, sorry for the noise

Reply:
https://github.com/llvm/circt/commit/5ce156846ac51948253d06fa11cbf4632160c432

Reply:
Thanks!